vignettes/intro_tutorial.qmd

```diff
@@ -338,82 +319,70 @@ with(
 )
 ```
 
+**tinyplot** also supports special types to fit models and display their predictions, along with confidence intervals. Here's an example where we fit a logistic regression model:
+
+```{r ribbon_pred}
+with(aq,
+  tinyplot(
+    I(Temp > 80) ~ Wind,
+    type = type_glm(family = binomial))
+)
+```
+
+We will see examples of more plot types below, including a other model prediction types. Please also take a look at [the dedicated Plot Types vignette](types.qmd) for explicit details on all of the different plot types that tinyplot supports, as well as how to create your own custom types.
+
 ## Facets
 
 Alongside the standard "by" grouping approach that we have seen thus far,
 **tinyplot** also supports faceted plots. Mirroring the main `tinyplot` function, the
 `facet` argument accepts both atomic and formula methods.
 
 ```{r facet_simple}
-with(
-  aq,
+with(aq,
   tinyplot(
-    x = Day, y = fit,
-    ymin = lwr, ymax = upr,
-    type = "ribbon",
-    facet = Month, ## <= facet, not by
-    grid = TRUE,
-    main = "Predicted air temperatures"
-  )
+    Temp ~ Wind,
+    facet = Month,
+    type = type_lm())
 )
 ```
 
 By default, facets will be arranged in a square configuration if more than
 three facets are detected. Users can override this behaviour by supplying
 `nrow` or `ncol` in the "facet.args" helper function. (The margin padding
 between individual facets can also be adjusted via the `fmar` argument.) Note
-that we can also reduce axis label redundancy by turning off the plot frame.
 
 ```{r facet_nrow}
-with(
-  aq,
+with(aq,
   tinyplot(
-    x = Day, y = fit,
-    ymin = lwr, ymax = upr,
-    type = "ribbon",
+    x = Wind, y = Temp,
     facet = Month,
     facet.args = list(nrow = 1),
-    grid = TRUE, frame = FALSE,
-    main = "Predicted air temperatures"
-  )
+    type = type_lm())
```

Comment:
Q: Do you perfer to use `type = type_lm()` rather than `type = "lm"`. I don't have strong preferences, but lean slightly toward the former since I suspect users will find it cool that they can use the character shortcut. (Your call; I just wanted to flag.)